packages/react-components/src/components/VideoTile.tsx

```diff
@@ -52,7 +73,12 @@ export interface VideoTileProps {
   styles?: VideoTileStylesProps;
   /** user id for the VideoTile placeholder. */
   userId?: string;
-  /** Component with the video stream. */
+  /**
```

Comment:
I don't think we should do this, for a few reasons -

* This is a breaking change with respect to behavior (though not the TypeScript API). If Contoso was using a transparent injected tile before this change, their app will break.
* It is surprising that this `render*` method behaves differently from all the other `render` methods. Granted this is named `renderElement` as opposed to `onRender*` and is intended to be *always* used as opposed to overriding interanlly rendered UI. Still, this is surprising behavior in the context of our library.
* Most importantly though, the same effect should be achieavable by Contoso without the changed behavior: `renderElement` is just a `JSX.Element`, so as long as we provide the building blocks for rendering the placeholder, Contoso can overlay the video and provide that here. We already support `onRenderPlaceholder`, so we should anyway include a way for Contoso to easily render placeholder (i.e., another public component in the UI library). This case is easy as the placeholder is essentially just the `Persona` component.